plugins/funind/recdef.ml

```diff
@@ -1417,7 +1419,7 @@ let com_terminate
     nb_args ctx
     hook =
   let start_proof ctx (tac_start:tactic) (tac_end:tactic) =
-    let evd, env = Pfedit.get_current_context () in
+    let evd, env = Pfedit.get_current_context () in (* XXX *)
```

Comment:
@Matafou @forestjulien I'm not sure I follow the logic here: we pass the current proof state to `start_proof`? Can you explain a bit what this code is doing? Thanks!

Reply:
I'll leave this part as it is.